test/command/suite/select/function/string/string_slice/ascii_regex.test

```diff
@@ -0,0 +1,10 @@
+plugin_register functions/string
+
+table_create Memos TABLE_HASH_KEY ShortText
+
+load --table Memos
+[
+{"_key": "Groonga"}
+]
+
+select Memos   --output_columns '_key, string_slice(_key, "(roon)(g)", 2, "")'
```

Comment:
Could you create a subdirectory for each patterns:

 * `string_slice(ascii, nth)`: `test/command/suite/select/function/string/string_slice/nth/ascii/`
 * `string_slice(multibyte, nth)`: `test/command/suite/select/function/string/string_slice/nth/multibyte/`
 * `string_slice(ascii, pattern, nth)`: `test/command/suite/select/function/string/string_slice/pattern/ascii/`
 * ...